modules/nf-core/genescopefk/meta.yml

```diff
@@ -54,6 +54,10 @@ output:
       type: file
       description: GeneScope histogram summary
       pattern: "*_summary.txt"
+  - kmer_cov:
+      type: string
```

Comment:
It's always a float, isn't it ?
```suggestion
      type: float
```

Reply:
Yes, it is always float

Reply:
The groovy class is a String, but that string is always a float as that's what the format string prints.